digdag-docs/src/operators/require.md

```diff
@@ -71,3 +88,8 @@
   ignore_failure: true
   ```
 
+  require> evaluates *ignore_failure* at last of its process. If rerun_on is set and require> run new attempt, the result of new attempt is checked.
+
+## Notes
+- require> has been changed to ignore inherited *retry_attempt_name* parameter. 
+  `digdag retry ` command generates unique retry_attempt_name to run, but it is not passed to require>.
```

Comment:
Unnecessary space on the right of `digdag retry`

Reply:
Fixed it. Thanks!